madara/node/src/cli/l1.rs

```diff
@@ -33,27 +33,23 @@ pub struct L1SyncParams {
     pub l1_endpoint: Option<Url>,
 
     /// Fix the gas price. If the gas price is fixed it won't fetch the fee history from the ethereum.
-    #[clap(env = "MADARA_GAS_PRICE", long, alias = "gas-price")]
-    pub gas_price: Option<u64>,
+    #[clap(env = "MADARA_GAS_PRICE", long)]
+    pub gas_price: Option<u128>,
```

Comment:
this is l2 gas or l1 gas? l1 gas right? maybe we should clarify the fields names in chainconfigs/cli args etc